GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Tags.cs

```diff
@@ -138,7 +136,7 @@ private Nodes FillTagTree(IReadOnlyList<IGitRef> tags, CancellationToken token)
                 {
                     token.ThrowIfCancellationRequested();
 
-                    bool isVisible = !IsFiltering.Value || _refsSource.Contains(tag.ObjectId);
+                    bool isVisible = !IsFiltering.Value || (tag.ObjectId is not null && _refsSource.Contains(tag.ObjectId));
```

Comment:
Bug fix: If the `ObjectId` was null, it would throw when passed to the `Contains` method (null key exception).